Survey/Survey/Sources/Presentation/Views/Buttons/LoginButton.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  LoginButton.swift
+//  Survey
+//
+//  Created by Taher on 3/11/22.
+//
+
+import SwiftUI
+
+struct LoginButton: View {
+    var body: some View {
+        Text(LocalizedTextAssets.localizable.login_button())
```

Comment:
Please add a blank line after type definition 🙏

Reply:
update in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/41/commits/f10d87bdfe1267e19fe24bb94768bd66860444a7)